src/server/typingsInstaller/nodeTypingsInstaller.ts

```diff
@@ -1,4 +1,3 @@
-/// <reference path="typingsInstaller.ts"/>
 /// <reference types="node" />

```

Comment:
you do not need this one either. the tsconfig.json already has `types: [ "node" ]`